### DISCUSSION TITLE
本周五前提交的基本功能实现的ppt是需要重新做一个ppt吗

### DISCUSSION THREAD

**Discussion by syhien** [liusy58/NJU_AdvancedProgramming_SP22#140] [Q&A] [answered]
cslabcms:
> 将源文件压缩成zip文件提交，包括ppt， 文件名：学号.zip

请问包括的ppt是要 根据实际的基本功能重做ppt 还是 在第3周的那个ppt的基础上修改 ？

**Comment by binbin00** [comment, marked as answer]
根据实际的代码实现情况进行改进即可。

**Reply by huanhuan6666** [reply]
请问PPT具体需要写什么内容？需要写功能怎么实现的吗

**Reply by binbin00** [reply]
1. 最好有更细节的程序框架，[以前的学生画的，供参考(wps画的)](http://assets.processon.com/chart_image/5ac064d4e4b04a5e960ced4b.png)
2. 核心的功能和有难度的功能的实现细节。

**Reply by Price1999a** [reply]
> 1. 最好有更细节的程序框架，[以前的学生画的，供参考(wps画的)](http://assets.processon.com/chart_image/5ac064d4e4b04a5e960ced4b.png)
> 2. 核心的功能和有难度的功能的实现细节。

我画的 （好菜啊
